docs/running-node.md

```diff
@@ -342,3 +342,85 @@ As a general guidance you may consider to use only the peers you own and/or trus
   * *Notes*
     * `cosmovisor` service should be restarted on any configuration changes
     * in case of any IP filtering firewall rules they should be updated as well
+
+## Adding new full node to existing network
```

Comment:
Probably makes sense to create a separate doc with this info, as it's quite important. 
Also makes sense to mention:
1) Put a link to that new doc at the beginning of the `running-node`one. I think we need to explicitly mention (with bold), that `running-node` is for running nodes in a new fresh pool only.
2) Put a link to that new doc to README.md
3) Put a link to that new doc to https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/how-to.md#node-admin-instructions-setting-up-a-new-validator-node